public/js/editStudent.js

```diff
@@ -0,0 +1,45 @@
+const Url = window.location.href;
+const splitUrl = Url.split('/');
+const cohortId = splitUrl[splitUrl.length - 3];
+const editStudentButton = document.getElementsByClassName('editButton');
+const newUsername = document.getElementById('username');
+const newName = document.getElementById('name');
+let studentNameValue = '';
```

Comment:
Why `let`? You only have changed the value once on line 10, then never again. Just declare it on line 10 with it's value. Same with `gitHubUserNameValue`.

And agree with your teammates whether it is `github` or `gitHub` because this started to annoy me...

Reply:
here we declared it as empty string because we want to put the value after click , and because we use it in admin.js file which contain fetchApi function which fetch from github Api so if it get a result , it declare an object that need the studentNameValue and gitHubUserNameValue which have a value now ! .
got it ?

Reply:
Another turn around, you could say:
```js
const info = { studentNameValue: '', gitHubNameValue: '' };
```
Then edit this object as you wish.
@HemaSAli